ui/app/store/actions.js

```diff
@@ -538,12 +457,6 @@ function unlockAndGetSeedPhrase (password) {
   }
 }
 
-function revealSeedConfirmation () {
-  return {
-    type: this.REVEAL_SEED_CONFIRMATION,
```

Comment:
This action constant can be removed, along with the block for this in the `app` reducer, and the test code for that reducer.

Reply:
Fixed